### PR TITLE
refactor code - dataflow for 0.19 

### PR DESCRIPTION
**Pull request by lfunderburk**
remove obsolete input classes, incorporate latest replacements ensuring functionality 
[Guide  ](https://bytewax.io/guides/handling-missing-values)

### CODE REVIEW

**Comment by lfunderburk** [comment]
Hi Oli,

This PR handles another guide update for 0.19, appreciate review/comments

File: README.md [hunk 1]
```diff
@@ -25,34 +25,69 @@ numpy
...
+### Goal: generate a dataflow that will impute missing values in a stream of data of random integers and `nan` values.
+
+We can represent our dataflow - called `map_eg` through this diagram, in which the data flows through three key steps:
 
-Bytewax is based around the concept of a dataflow. A dataflow is made up of a sequence of operators that interact with data that is “flowing” through it.
+1. input
+2. stateful map
+3. output
+
+![](./diagram.png)
```
Comment (oli-kitty) [inline README.md]: Can it be replaced with a mermaid diagram?

Comment (lfunderburk) [reply README.md]: Asked David - he gave me a method to turn the dataflow into UML format, I then pasted that onto software to generate the diagram
Investigating if there is a way to turn it into a mermaid graph instead

Comment (lfunderburk) [reply README.md]: David has set up a PR to generate mermaid graphs from the dataflows
I added a temporary mermaid diagram generated with other software

**Review by oli-kitty — APPROVED** [review]

**Pull request merged**